### DISCUSSION TITLE
Restore From Backup Not Working

### DISCUSSION THREAD

**Discussion by kn4thx** [hassio-addons/addon-unifi#476] [Q&A]
When I start the add-on, there is an option to restore from backup. When I click that, I get a blank page. Is this not available on the add-on version?

**Comment by renewoensdregt** [comment]
> When I start the add-on, there is an option to restore from backup. When I click that, I get a blank page. Is this not available on the add-on version?

I have seen his too when I was testing. It's at the initial setup as during the upgrade it keeps your data. 

As workaround, have you tried to set it up from scratch and from within the application restore a backup?

**Comment by renewoensdregt** [comment]
I did check on an earlier version and there it does come up, so it seems to be specific to this version. Not sure I have the skills to fix it though. Did you try the workaround?

**Reply by renewoensdregt** [reply]
Tried again today and it does come up for me now, so maybe a performance issue and wait a bit or use the workaround.
